tools/lint/src/main/java/com/grab/lint/Sanitizer.kt

```diff
@@ -24,47 +22,25 @@ class Sanitizer(
         tmpFile.toFile().copyTo(out, overwrite = true)
     }
 
-    /**
-     * It seems we can't simply rely on PWD to find the execroot as Lint baseline output may contain sandbox realized paths in them which
-     * expands symlinks. For example, if the file is coming from another target's output it might contain a different sandbox root.
-     * Especially the sandbox number might be different.
-     *
-     * To solve this we check for sandbox path and prepare a [Regex] that can replace any sort of path sandbox or not.
-     */
-    private val rootRegex: Regex by lazy {
-        val pwd = env.pwd
-        val currDirName = File(pwd).name
-        val regex = "((-sandbox|_bazel_root)/(.*?)/execroot/$currDirName)|(/../../../../../)".toRegex()
-        val sandboxDir = regex.find(pwd)?.groupValues?.firstNotNullOfOrNull { it.toIntOrNull() }
-        val pattern = when {
-            sandboxDir != null -> pwd.replace("/$sandboxDir/", "/(.*?)/")
-            else -> pwd
+    //message="Missing density variation folders in `../../../../../../private/var/tmp/_bazel_mohammad.khaleghi/f9bf0b8d7b1d5760a1f747cde05d58d9/execroot/pax_android/food-screen/food-cart/src/main/res`: drawable-hdpi, drawable-mdpi, drawable-xhdpi">
+    private fun sanitized(line: String): String {
+        var target = "file=\""
+        var targetIndex = line.indexOf(target)
+        if (targetIndex == -1 && line.contains("message=\"")) {
+            target = "\'"
+            targetIndex = line.indexOf(target)
         }
-        (pattern.removePrefix("/") + "/").toRegex()
-    }
-
-    private fun sanitize(line: String, calcExecRoot: Regex): String {
-        val target = if ("file=\"" in line) {
-            "file="
-        } else if ("message=\"" in line) {
-            "message="
+        return if (targetIndex != -1) {
+            line.substring(0, targetIndex + target.length) +
+                    line.substring(targetIndex + target.length)
+                        .replace(System.getenv(PWD), "\$$PWD")
```

Comment:
`System.getenv(PWD)` consider accessing this via `env.pwd` as before, it was used for substituting in tests.